source/documentors/references/templates/doc_concept_template.rst

```diff
@@ -57,20 +57,24 @@ RST Template
 	External Links
 	--------------
 
-	You can use `inline links <https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#linking>`_ or `indirect links`_. Indirect links are useful if you want to link to the same thing multiple times, or if the URL is really long and you want the raw RST to read more cleanly.
+.. warning::
+   Usage of ``:doc:`` is an antipattern. It is fragile and prone to breaking cross references when docs are moved or renamed.
 
-	.. _Indirect links: https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#linking
+   Adding in ``.. _reference:`` syntax to files and headings means cross-references can instead be made with the ``:ref:`` directive, which will be more robust to docs refactorings.
+	You can use `inline links <https://docs.openedx.org/en/latest/documentors/references/quick_reference.html#linking>`_ or `indirect links`_. Indirect links are useful if you want to link to the same thing multiple times, or if the URL is really long and you want the raw RST to read more cleanly.
+
+	.. _Indirect links: https://docs.openedx.org/en/latest/documentors/references/quick_reference.html#linking
 
 	Internal Links
 	--------------
 
-	Link to :doc:`doc_how_to_template` in the same folder or :doc:`../quick_reference_rst` in a different
+	Link to :doc:`doc_how_to_template` in the same folder or :ref:`Documentation Syntax Reference` in a different
 	folder.
 
 	By default, it will use the title of the doc as the link text but you can override that with :doc:`other text </doc_how_to_template>` if you want.
```

Comment:
```suggestion
	By default, it will use the title of the doc as the link text but you can override that with :doc:`other text <doc_how_to_template>` if you want.
```